### ISSUE TITLE
Translated only the diff elements

### ISSUE THREAD

**Issue by timheuer** [IEvangelist/resource-translator#4]
Maybe a tall task, but given how the Azure service works (word count), you will be over-translating after the first run.  Consider only somehow translating the diff/change.

I realize this is a bigger ask as you're basically just translating all of them and emitting files.

**Comment by IEvangelist** [comment]
Oh, I like this! I'm curious though how to do that. I suppose I could look into using something like this:

```typescript
import { context } from '@actions/github';'

const octokit = await context.getOctokit(token);
```

Then, I could [use it](https://github.com/actions/toolkit/tree/main/packages/github#usage) to get information about the pull request that caused the action to run, and use the deltas as input to translate. That does change a lot though, but I like it.

**Comment by IEvangelist** [comment]
Fixed in release [v2.0.3](https://github.com/IEvangelist/resource-translator/releases/tag/v2.0.3)
